.github/workflows/automated.yaml

```
@@ -0,0 +1,63 @@
+name: Release from React Native
+
+on:
+  repository_dispatches:
+    types: [release]
+
+# NOTE: Changes here need to be reflected in release.yaml
+jobs:
+  publish_template_automation:
+    runs-on: ubuntu-latest
+    env:
+      VERSION: ${{ github.events.client_payload.version }}
+    steps:
+      - name: Safeguard against branch name
+        run: |
+          if [[ "$BRANCH" != *-stable ]]; then
+            echo "Error: This workflow can only be executed from a branch ending with '-stable'."
+            exit 1
+          fi
+      - name: Checkout
+        uses: actions/checkout@v4.1.1
+      - name: Move to MAJOR.MINOR-stable or create if it doesn't exist
+        run: |
+          VERSION=$(grep -E '\d+\.\d+' <<< "$VERSION" | awk '{ print "branch="$0"-stable" }')
+          if ! git ls-remote -q --exit-code --heads origin $VERSION > /dev/null; then
+            echo "Creating missing release branch \"$VERSION\""
+            git checkout -B "$VERSION"
+            git push --set-upstream origin "$VERSION"
+          else
+            git checkout "$VERSION"
+          fi
+      - name: Setup node.js
+        uses: actions/setup-node@v4.0.0
+        with:
+          node-version: 18
+          registry-url: 'https://registry.npmjs.org'
+      - name: Update versions to input one
+        run: node ./scripts/updateTemplateVersion.js "$VERSION"
+      - name: Update template/package.json to nightly react-native + @react-native
+        run: node ./scripts/updateReactNativeVersion.js "$VERSION"
+      - name: Create corresponding commit & git tag
+        run: |
+          git config --global user.name 'React Native Bot'
+          git config --global user.email 'bot@reactnative.dev'
+          git commit -am "Bumping template to $VERSION"
+          git push
+          git tag $VERSION
```

with or without the leading `v`?

```suggestion
git tag "$VERSION"
```

---

Good spot.